server/db/models/order.js

```diff
@@ -0,0 +1,25 @@
+const Sequelize = require('sequelize')
+const db = require('./db')
+
+const Order = db.define('order', {
+  userId: {
```

Comment:
Sorry - I don't think we need userId or sessionId on the model - these will be defined through associations, right?